src/js/cash-bail-and-race.js

```diff
@@ -152,8 +152,8 @@ const createBailRaceAmountTable = () => {
   const initSort = { col: 4, dir: -1 }; // initially sort by difference
   const stateData = [
     "Pennsylvania",
-    "$36,202",
-    "$26,868",
+    "$36.2K",
+    "$26.8K",
```

Comment:
26.9?

Reply:
Lol yes, but the data's might be changed anyway so I'll leave it